lib/tasks/database_updates/backfill_publisher_statuses.rake

```diff
@@ -0,0 +1,19 @@
+namespace :database_updates do
+  task :backfill_publisher_statuses => :environment do
+    admin = Publisher.find_by(email: Rails.application.secrets[:zendesk_admin_email])
+    admin = Publisher.first
+
+    publishers = Publisher.all.
+        joins('LEFT OUTER JOIN publisher_status_updates ON publisher_status_updates.publisher_id = publishers.id').
+        where('publisher_status_updates.publisher_id is NULL').
+        order(created_at: :desc)
+
+    publishers.find_each do |publisher|
+      note = publisher.notes.create(note: "Backfilling active status", created_by_id: admin.id)
+      publisher.status_updates.create(status: PublisherStatusUpdate::ACTIVE, publisher_note: note)
+      puts 'Created'
```

Comment:
Would you put in the publisher_id so we know where it stopped if it does get stop in the middle?

Also would be good for the script to be retried with parameters.

Reply:
https://github.com/brave-intl/publishers/pull/2703/commits/637f91ebcdf491404b19814e9cf79aae19b40bf7

I think we'll be okay with re-running the script without passing in any arguments. The query will return publishers who don't currently have a status. So re-running this script will only target the publishers not yet touched by the script.